calcom/cal.com
# fix: New Booking on Group Event shows other guest name on confirmation

## What does this PR do?
I have changed the title structure of booking for group event with multiple seats. Now for group event it be like [ event name ] by [ host name ] in place of [ event name ] between [ host name ] and [ attendee ]. This will fix the issue of showing other attendee name while booking a group event by other guest user, earlier the title was created based upon the name of first attendee now it will be a fixed title only for group event else will be same as earlier. 




Fixes #9292 

Before - 
![before](https://github.com/calcom/cal.com/assets/77577810/8cb90b1d-48e8-4bcf-8343-960ed8415711)

After -
![after](https://github.com/calcom/cal.com/assets/77577810/95cc02b7-5228-4ef1-a635-093255842afc)

**Environment**: Staging(main branch) / Production

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
Create group event booking and book the event as guest, you'll see change in title (what) of the scheduled meeting
